libusb/core.c

```diff
@@ -1440,6 +1441,23 @@ int API_EXPORTED libusb_open(libusb_device *dev,
 	return 0;
 }
 
+/** \ingroup libusb_dev
+ * Checks if a device handle is still connected to the system.
+ *
+ * This is a non-blocking function; no requests are sent over the bus.
+ *
+ * \param dev_handle device handle to be checked.
+ * \returns 0 on if the device is connected
```

Comment:
I think the "on" is leftover from the previous edit?

Reply:
Fixed